### ISSUE TITLE
Clicking the "Add topics" link at a GitHub repository starts loading animation, but nothing happens

### ISSUE THREAD

**Issue by MikhailTymchukDX** [DevExpress/testcafe#1480]
### Are you requesting a feature or reporting a bug?
Bug.

### What is the current behavior?
When TestCafe clicks the "Add topics" link on a GitHub repository, loading animation is playing indefinitely.

### What is the expected behavior?
When TestCafe clicks the "Add topics" link on a GitHub repository, a page loads an input element after animation is played.

#### Provide the test code and the tested page URL (if applicable)

The tested page URL: https://github.com/MikhailTymchukDX/E1016-ASPxGridView-How-to-customize-detail-buttons

Test code

```js
import { Selector } from 'testcafe';

fixture `Topic add bot`
    .page `https://github.com/MikhailTymchukDX/E1016-ASPxGridView-How-to-customize-detail-buttons`;

test('Add .NET topic', async t => {    
    const signInLink1 = await Selector('.site-header-actions').find('a').withText("Sign in");

    await t
        .click(signInLink1)
        .typeText("#login_field", "<your login here>")
        .typeText("#password", "<your password here>");

    const signInSubmit = await Selector("input[value='Sign in']");
	await t.click(signInSubmit);
	const addTopics = await Selector("button").withText("Add topics");
    await t.click(addTopics).wait(5000);        
});
```

### Specify your

* operating system: Windows 8.1 64-bit
* testcafe version: 0.15.0
* node.js version: 7.4.0

**Comment by AlexanderMoskovkin** [comment]
It's related to the `testcafe-hammerhead`, so I've created an issue: https://github.com/DevExpress/testcafe-hammerhead/issues/1129

**Comment by AlexanderMoskovkin** [comment]
fixed in `testcafe@0.16.0-alpha3`
